### DISCUSSION TITLE
HP : Le numérique, ce n’est pas que du vent

### DISCUSSION THREAD

**Discussion by margaux-escande** [cnumr/EcoIndex#150] [Bug]
Sur la page d'accueil il y a deux fois en sous titre "Le numérique, ce n’est pas que du vent" 
Je ne sais pas si c'est normal ?

**Comment by DocRoms** [comment]
Effectivement. J'imagine que ça n'était pas dans les maquettes 😹 .

@aureliebaton , @Bastou vous connaissez le sous-titre qu'il aurait fallu mettre après `Pourquoi c’est important ?` ou après `Sa contribution à l’empreinte de l’humanité est loin d’être négligeable` sur la page d'accueil ?

**Comment by aureliebaton** [comment]
Oui c'est un bon point, on peut le retirer sous " Sa contribution à l’empreinte de l’humanité est loin d’être négligeable". Ca donnerait comme les maquettes 
<img width="445" alt="Screenshot 2022-09-20 at 13 56 25" src="https://user-images.githubusercontent.com/85348619/191251578-525efdfb-82c0-464c-b82b-ef30807ed704.png">

<img width="528" alt="Screenshot 2022-09-20 at 13 56 43" src="https://user-images.githubusercontent.com/85348619/191251606-990e42ae-f28e-4ea3-84a4-850452b493a3.png">

@Bastou @yaaax y a moyen de mettre à jour ? Merci !!

**Comment by yaaax** [comment]
Je viens de corriger ;)
